springmvc-raml-parser/src/main/java/com/phoenixnap/oss/ramlapisync/generation/rules/spring/SpringControllerDecoratorRule.java

```diff
@@ -63,7 +68,18 @@
  * @author armin.weisser
  * @since 0.4.1
  */
-public abstract class SpringControllerDecoratorRule implements Rule<JCodeModel, JDefinedClass, ApiResourceMetadata> {
+public abstract class SpringControllerDecoratorRule implements ConfigurableRule<JCodeModel, JDefinedClass, ApiResourceMetadata> {
+    public static final String CALLABLE_RESPONSE_CONFIGURATION = "callableResponse";
+
+    private boolean callableResponse;
```

Comment:
Can you please initialize configuration variables?
Even though they default to false, it's not super readable this way.